src/components/Article.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({
+  title,
+  text,
+  date,
+}) => (
+  <article>
+    <h1>
+      {title}
+    </h1>
+
+    <p>
+      {text}
+    </p>
+
+    <span>
+      {date}
+    </span>
+  </article>
+);
```

Comment:
```suggestion
const Article = ({ title, text, date }) => (
  <article>
    <h1>{title}</h1>
    <p>{text}</p>
    <span>{date}</span>
  </article>
);
```

Reply:
no need to put every prop on the new line, although you can if there're a lot of them)